stablehlo/tests/ops_stablehlo.mlir

```diff
@@ -2897,6 +2876,114 @@ func.func @reverse_i2(%operand: tensor<3x2xi32>) -> tensor<3x2xi32> {
 
 // -----
 
+// CHECK-LABEL: func @while
```

Comment:
Just curious to know, if these tests duplicate of the ones added in verify_while.mlir?
In any case, we can keep  them in its dedicated file `verify_while.mlir`?